micro-rdk/src/common/conn/server.rs

```diff
@@ -26,7 +26,7 @@ use crate::{
 
 use async_io::Timer;
 use futures_lite::prelude::*;
-use futures_lite::{future::Boxed, ready, Future};
+use futures_lite::{future::Boxed, ready};
```

Comment:
`Future` is already in via `prelude::*` so there was a warning I cleaned up.

Reply:
While building or returned by clippy?

Reply:
While building native on my mac.